mail/urlscan/Portfile

```diff
@@ -17,13 +17,16 @@ long_description    ${name} lets you easily browse URLs contained in an email\
 
 github.tarball_from archive
 
-python.default_version 39
+python.default_version 312
+python.versions 39 310 311 312
 
-checksums           rmd160  439e24ccb82619affe45c22bbca2272d466b875a \
-                    sha256  db76f61966431a8938adc998aaa2b30be54f9b1ff2a4c0064e9915df2fb6b996 \
-                    size    32010
+checksums           rmd160  d8842153eacd06a2085f2e44c1c5e85b5af4b1d2 \
+                    sha256  f9c0b6067b1327059bf6298bd6f4834148a3a71bb9864943c0cb1d6872de5c6c \
+                    size    35837
 
 depends_build-append \
-                    port:py${python.version}-setuptools
+                    port:py${python.version}-setuptools \
+                    port:py${python.version}-hatchling \
```

Comment:
please set "python.pep517_backend hatch" instead of explicitly adding this dependency; also are you sure "setuptools" is still needed, it shouldn't...